vulcanai/run.py

```diff
@@ -0,0 +1,47 @@
+from torch.utils.data import DataLoader, Subset
+from vulcanai.datasets import TabularDataset
+from vulcanai.models import ConvNet, DenseNet
+import sys
+sys.path.append('../')
+from models.metrics import conduct_sensitivity_analysis
+
+def my_test_dataset():
+    """Create a dataset by importing from the test csv"""
+    fpath = "/Users/snehadesai/Documents/Vulcan/vulcanai/tests/datasets/test_data/birthweight_reduced.csv"
+    return TabularDataset(
+        data=fpath,
+        label_column="id",
+        na_values='Nan'
+    )
+
+def tabular_dataset_train_loader(my_test_dataset):
+    """Test csv data pytorch dataloader object."""
+    test_train = Subset(my_test_dataset,
+                        range(len(my_test_dataset)//2))
+    return DataLoader(test_train, batch_size=2)
+
+def dnn_class_single_value_2():
+    """DenseNet with prediction layer."""
+    return DenseNet(
+        name='dnn_class',
+        in_dim=(12),
+        config={
+            'dense_units': [100, 50],
+            'dropout': 0.5,
+        },
+        num_classes=22
+    )
+
+
+
+def run_sensitivity_analysis(dnn_class_single_value_2,
+                              tabular_dataset_train_loader):
+    conduct_sensitivity_analysis(dnn_class_single_value_2,
```

Comment:
A thought: Maybe we should return some JSON object along with saving the csv in case we want to programmatically pass the results through an API? @nirtiac I'd also like to get your thoughts on this

Reply:
yes I think we should. we need to standardize this interface though as all our metric functions return something else. so for now just return a python dictionary and save the file but it may be changed in the future.

Reply:
How should this dict be arranged? Right now, it returns a df with headers {Feature, Value, Number of examples classified as class 0, ... 1, all the way to n (num classes)}. List of dicts per example with these headers as keys? Or something else?

Reply:
oh hmmmm, a dataframe may indeed be more appropriate then. just specify it in the function documentation.